vault.gemspec

```diff
@@ -19,6 +19,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency "aws-sdk"
```

Comment:
We keep the client libraries to a zero runtime dependency requirement to make them fast, flexible, and unlikely to conflict with other libraries. Given how heavy the aws-sdk is, I am very opposed to including it as a dependency.